src/repositories/highscoreRepository.js

```diff
@@ -0,0 +1,26 @@
+module.exports = db => {
+  return {
+    async getHighScores() {
+      try {
+        const { rows } = await db.query(
+          `SELECT name, score FROM "HighScores" ORDER BY SCORE ASC LIMIT 10`
+        );
+        return rows.map(({ name, score }) => ({ name, score }));
+      } catch (err) {
+        console.log(err.stack);
+      }
+    },
+    async addHighScore(name, score) {
+      try {
+        const query = {
+          text: `INSERT INTO "HighScores"(name, score) VALUES($1, $2)`,
+          values: [name, score]
+        };
+
+        db.query(query);
```

Comment:
I have checked in node-postgres WIKI that the library handle SQL injections via the parameterized query (that we are using)

> 8. Does node-postgres handle SQL injection?
> Absolutely! The parameterized query support in node-postgres is first class. All escaping is done by the postgresql server ensuring proper behavior across dialects, encodings, etc... For example, this will not inject sql:
> 
> ```
> client.query("INSERT INTO user(name) VALUES($1)", ["'; DROP TABLE user;"], function (err, result) {
>   // ...
> });
> ```

I have tried to inject SQL with no success